pipeline/pipeline.go

```diff
@@ -487,24 +530,27 @@ func (p *Pipeline) maintenance() {
 		p.antispamer.maintenance()
 		p.metricsHolder.maintenance()
 
-		totalCommitted := p.totalCommitted.Load()
-		deltaCommitted := int(totalCommitted - lastCommitted)
-
-		totalSize := p.totalSize.Load()
-		deltaSize := int(totalSize - lastSize)
+		// set new values and get the diffs
```

Comment:
maybe move this in separate function?

Reply:
done, please let me know if it looks okay